api/alert_channels_jira_cloud_server.go

```diff
@@ -24,6 +24,34 @@ import (
 	"strings"
 )
 
+type jiraIssueGrouping int
+
+const (
+	NoJiraIssueGrouping jiraIssueGrouping = iota
+	EventsJiraIssueGrouping
+	ResourcesJiraIssueGrouping
+)
+
+var JiraIssueGroupings = map[jiraIssueGrouping]string{
+	NoJiraIssueGrouping:        "",
+	EventsJiraIssueGrouping:    "Events",
+	ResourcesJiraIssueGrouping: "Resources",
+}
+
+var JiraIssueGroupingsSurvey = map[string]jiraIssueGrouping{
+	"None":      NoJiraIssueGrouping,
```

Comment:
Here is good 😂 - so I think we /should/ change it
```suggestion
	"None":      NoneJiraIssueGrouping,
```

Reply:
Fixed.